### PR TITLE
Resolved multiple issues found during testing, added backwards compatibility

### PR DESCRIPTION
**Pull request by EasyAsABC123**
Bug Fixes for:
https://github.com/opscode-cookbooks/iis/issues/95
https://github.com/opscode-cookbooks/iis/issues/98
https://github.com/opscode-cookbooks/iis/issues/100
https://github.com/opscode-cookbooks/iis/issues/99
https://github.com/opscode-cookbooks/iis/issues/101

had to remove unneeded variables
incorrect variables
add dependencies
bumped version
added backwards compatibility for chef-client < 12.x.x Chef::Util::PathHelper
fixed bindings

### CODE REVIEW

**Comment by EasyAsABC123** [comment]
Paging @adamedx, @btm, @jdmundrawala and @jtimberman

Ready for Code Review from a testing standpoint.

**Comment by smurawski** [comment]
Working through it now @EasyAsABC123

**Comment by jdmundrawala** [comment]
I think @smurawski said he was going to be taking a look at PRs here today

**Comment by EasyAsABC123** [comment]
@jdmundrawala thanks
@smurawski  Thanks, I forgot who I spoke to yesterday. :+1:

File: metadata.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@
 license          "Apache 2.0"
 description      "Installs/Configures Microsoft Internet Information Services"
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          "3.0.0"
+version          "3.0.9"
 supports         "windows"
 depends          "windows", ">= 1.34.6"
+depends          "chef-client", ">= 3.7.0"
```
Comment (smurawski) [inline metadata.rb]: Why the additional dependency? I don't see OpsCode::ChefClient::Helpers or any of the recipes used.

Comment (EasyAsABC123) [inline metadata.rb]: I thought that was required for Chef::Util::PathHelper, is this incorrect?

**Comment by smurawski** [comment]
@EasyAsABC123 Would you mind fixing the encoding on README.md?  It's set as UTF-8 and showing as a binary file.

**Comment by EasyAsABC123** [comment]
@smurawski I Saved the file as UTF-8 but it isn't showing as a change occurring.

**Comment by smurawski** [comment]
Yeah, github can be picky about that.. Re-encode as ASCII or make sure there is no BOM?

File: metadata.rb (hunk 1 above)
Comment (smurawski) [inline metadata.rb]: No, PathHelper is in core chef-client, not the cookbook.

**Comment by smurawski** [comment]
Otherwise, it's looking pretty good.  So, remove the dependency on the chef-client cookbook and fix the file encoding on the README.md and I'll be happy to merge it.

**Comment by EasyAsABC123** [comment]
@smurawski I think I fixed the issue of encoding, also removed the dependency

**Pull request merged**